pkg/kubelet/cm/device_plugin_handler.go

```diff
@@ -251,7 +324,7 @@ type checkpointData struct {
 
 // Checkpoints device to container allocation information to disk.
 func (h *DevicePluginHandlerImpl) writeCheckpoint() error {
-	filepath := h.devicePluginManager.CheckpointFile()
+	filepath := h.checkpointFile()
```

Comment:
Is there a need for a function at this point shouldn't this be a const

Reply:
Not sure about the meaning of "const". You mean change the  socket file name from a magic string to const ?
The socketdir is variable anyway

Reply:
right just went trough the declaration again. Forgot that `h.socketdir` can be different :)